Documentation/operations/metrics.rst

```diff
@@ -530,6 +530,32 @@ Name                             Labels                                   Descri
 Options
 """""""
 
+This metric supports :ref:`Context Options<hubble_context_options>`.
+
+``flows-to-world``
+~~~~~~~~~~~~~~~~~~
+
+This metric counts flows that contain ``reserved:world`` label in the destination
+identity. By default, it counts dropped flows if and only if the drop reason is
+``Policy denied``. Set ``any-drop`` option to count all dropped flows.
+
+================================ ======================================== ========================================
+Name                             Labels                                   Description
+================================ ======================================== ========================================
+``flows_to_world_total``         ``protocol``, ``verdict``                Total number of flows to reserved:world.
```

Comment:
I find the metric name confusing. It is a total count of **denied** flows to world but the name suggests it's a count of flows to world. What about something like `dropped_flows_to_world_total` which would count any dropped flows regardless of the reason by default, with an option key `denied` which would only count dropped flows that were actually denied?

Reply:
OK, re-reading, it seems that this is a metric that counts all flows to world after all. However, I somehow read the doc as "all flows to world which are dropped by policy". I didn't have a :coffee: this morning yet so it could be me. If not, please, improve the doc on this point :)